### PR TITLE
a working carousel

### PR DESCRIPTION
**Pull request by minaorangina**
Replaced the image carousel slider with one that works better.

### CODE REVIEW

**Comment by minaorangina** [comment]
oct-28-2018 00-22-39

File: about.html [hunk 1]
```diff
@@ -110,8 +112,8 @@ <h3 class="header center green-text">ABOUT US</h3>
       </p>
       <div class="photo-container">
 
-        <div class="prev-photo carousel-button">👈</div>
-        <div class="siema">
+        <!-- <div class="prev-photo carousel-button">👈</div> -->
```
Comment (claireinez) [inline about.html]: Probably best just to get rid of these commented out buttons?

File: about.html [hunk 2]
```diff
@@ -190,10 +192,20 @@ <h5 class="white-text">Contact Us</h5>
 
 
   <!--  Scripts-->
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/tiny-slider/2.8.7/min/tiny-slider.js"></script>
   <script>
-    const mySiema = new Siema({loop: true, duration: 400})
-    document.querySelector('.prev-photo').addEventListener('click', () => mySiema.prev())
-    document.querySelector('.next-photo').addEventListener('click', () => mySiema.next())
+    const slider = tns({
```
Comment (claireinez) [inline about.html]: I'm not sure of our browser support but this should maybe be a `var` just in case

File: about.html [hunk 3]
```diff
@@ -190,10 +192,20 @@ <h5 class="white-text">Contact Us</h5>
...
+      container: '.slider',
+      slideBy: 'page',
+      speed: 500,
+      controls: false,
+      nav: false,
+      autoplay: true,
+      autoplayTimeout: 3200,
+      autoplayButtonOutput: false
+    })
+    console.log(slider)
```
Comment (claireinez) [inline about.html]: Don't need the `console.log` 😃

**Review by claireinez — COMMENTED** [review]
Just a few comments!

File: about.html (hunk 3 above)
Comment (minaorangina) [reply about.html]: Ah, missed that!

**Review by claireinez — APPROVED** [review]

**Pull request merged**